src/ApiVersioningExtension/ApiVersioningExtension/Extensions/ApiVersioningDependencyInjectionExtension.cs

```diff
@@ -55,23 +55,23 @@ private static void AddApiVersioning(IServiceCollection services, ApiVersioningC
 
             #region Default Version Parsing
 
-            if (!string.IsNullOrEmpty(config.DefaultApiVersion))
+            ApiVersion defaultVersion = ApiVersion.Default;
+            if (!string.IsNullOrWhiteSpace(config.DefaultApiVersion))
             {
-                var validApiVersion = ApiVersion.TryParse(config.DefaultApiVersion.Replace("v", ""), out ApiVersion defaultVersion);
-                if (!validApiVersion)
+                if (!ApiVersion.TryParse(config.DefaultApiVersion.Replace("v", ""), out defaultVersion))
                     throw new ArgumentException($"Not valid version found({config.DefaultApiVersion}). Usage Example: 1.0 or 1 or v1.0 or v1");
-
-                opt.DefaultApiVersion = defaultVersion; //ApiVersion.Default;
             }
 
+            opt.DefaultApiVersion = defaultVersion!;
```

Comment:
we don't want to specify the default version unless it's set with config.DefaultApiVersion. So this piece of code must be in the if condition